model_analyzer/output/output_table.py

```diff
@@ -33,14 +33,26 @@ class OutputTable:
     with headers rows
     """
 
-    def __init__(self, headers):
+    def __init__(self, headers, column_widths=None):
```

Comment:
It would be great if we can determine the column_widths inside this function based on the content and header without having to determine it from outside.